examples/gltf/app.js

```diff
@@ -371,8 +373,7 @@ export class DemoApp {
     });
   }
 
-  onRender({gl, time, width, height, aspect}) {
-    gl.viewport(0, 0, width, height);
+  onRender({gl, time, aspect, vrViewMatrix, vrProjectionMatrix}) {
```

Comment:
Idea: Should we just call these `viewMatrix` and `projectionMatrix` and have the base `AnimationLoop.onRender()` supply them (as identity matrices)?

That way all demos can be written to use them and transparently work with both animation loop classes?

Reply:
ok